### ISSUE TITLE
Bad VDC

### ISSUE THREAD

**Issue by davidcharlessmith516** [thomastech/PixelRadio#12]
I continue to receive a 'BAD-VDC' message in the Radio Status box, although the device appears to broadcast properly.

![image](https://github.com/thomastech/PixelRadio/assets/59949483/e9b7eaca-1027-4329-b535-a8258d139314)

I have tried three different 12V power adapters, 500mA, 1A & 2A and they all show roughly the same numbers in the diagnostics.

![Screenshot 2024-03-13 134310](https://github.com/thomastech/PixelRadio/assets/59949483/8a4e8905-98a5-4ed7-9c32-a93c414e7230)

Is there some physical issue with the board that I can troubleshoot? Do I have an incorrect setting somewhere (it's been like this since I first powered it on).

**Comment by thomastech** [comment]
The displayed RF PA Voltage measurement is wrong. Should be 9VDC. That is the reason for the BAD-VDC message.

Confirm that jumper J9 is installed. Re-solder the connections to this.
Confirm that you have approx 9VDC across C20.
Confirm that you have approx 9VDC across C26.

If all those things are correct then there is likely a bad solder connection or incorrect component at R8, R9 & C4. And check pin 1 of the MCU1 and its socket.

- Thomas

**Comment by davidcharlessmith516** [comment]
Thank you for responding so quickly. I swapped out the jumper for a different one and have the same BAD-VDC. I will search for my multimeter tomorrow to check the voltages, as the rest of the house is asleep right now. In the meantime, I will also share what may be another cause of the issue:

![IMG_20240314_005643370](https://github.com/thomastech/PixelRadio/assets/59949483/7f7b6df6-e9e6-484b-a198-6575dc82e049)

In trimming my solder joints, I scraped the board a little bit. Sure enough, this is right in between R8 and R9 where you guessed there could be an issue. In the photo, R8 is on the right and R9 is on the left. Not sure if this is causing the issue and/or if it is repairable if that's the case, but since it is right in the neighborhood of the components you said could have an issue, I thought I'd mention it.

I'll let you know what I come up with on the voltages when I find my multimeter.

Thanks again!

**Comment by davidcharlessmith516** [comment]
I had roughly 8.95V across both C20 and C26.
I confirmed continuity across J9 with the jumper installed.
I still had the BAD-VDC message and the same low RF PA Voltage

I went through and resoldered C20, C26, R8, R9 and C4. 
This resolved the issue. I now have no error message and 8.8 RF PA Voltage

![image](https://github.com/thomastech/PixelRadio/assets/59949483/feab70d9-c4e2-490d-8fb8-7d8ac7a1ba05)

![image](https://github.com/thomastech/PixelRadio/assets/59949483/f72b4429-672a-4063-b5f3-3e9b4038507b)


Thank you for your guidance. This is a very neat and extremely useful project that you came up with and I appreciate that you continue to maintain and support it.
